eclipse-collections/src/main/java/org/eclipse/collections/impl/list/fixed/QuadrupletonList.java

```diff
@@ -145,6 +146,12 @@ public T getOnly()
     }
 
     @Override
+    public Optional<T> getOnlyOptional()
```

Comment:
Of course. I just mechanically added `getOnlyOptional` to any class with `getOnly`. Let me check were in the class hierarchy `getOnlyOptional` really belongs. I will update the pull request. Thanks!